share/shwap/row.go

```diff
@@ -240,3 +240,70 @@ func toRowSide(s string) RowSide {
 		panic("invalid row side")
 	}
 }
+
+var codec = share.DefaultRSMT2DCodec()
+
```

Comment:
Any changes or just copy? Can't see the diff in gh

Reply:
Slightly prefer to keep it as separate file as it used to be.

Reply:
shwap folder contains containers so I'm afraid that the axis half could be treated as a separate container.

Reply:
Right now It is in row container file, which is more confusing and way harder to find